### ISSUE TITLE
Can we get the tabbar to the right ?

### ISSUE THREAD

**Issue by Frekansapp** [DaKaZ/react-native-vertical-tab-view#5]
Can we get the tabbar to the right ? Can we scroll vertically ?

**Comment by DaKaZ** [comment]
@Frekansapp for the first one

> Can we get the tabbar to the right 
this is easy, just use the `tabBarPosition="bottom"` prop.  In Vertical, top === left, bottom === right

> Can we scroll vertically
The tab bar itself should sroll vertically, but the contents scroll left/right only

**Comment by Frekansapp** [comment]
@DaKaZ Do you have the chance to add vertical scrolling i think it is a important feature. I try to implement it but i couldn't succeed.

**Comment by DaKaZ** [comment]
Are you talking about vertical scrolling of the main content window instead of left/right scrolling?  I actually had implemented that originally when this was part of the RNTV project, I could probably address.  I need to do an a major update to support v2 of the RNTV anyway.... It might take a little while though as this feature isn't that important to me personally.

**Comment by Frekansapp** [comment]
> 
> 
> Are you talking about vertical scrolling of the main content window instead of left/right scrolling? I actually had implemented that originally when this was part of the RNTV project, I could probably address. I need to do an a major update to support v2 of the RNTV anyway.... It might take a little while though as this feature isn't that important to me personally.

Yes, I think it is very useful if you can activate this feature by passing props. it doesn't matter when it takes time. I will wait. RNTV v2 have problems by the way can you look at the [this](https://github.com/react-native-community/react-native-tab-view/issues/712) and [this](https://github.com/react-native-community/react-native-tab-view/issues/717) issue about RNTV?
